tests/gold_tests/pluginTest/TSVConnFd/CMakeLists.txt

```diff
@@ -15,9 +15,12 @@
 #
 #######################
 
-add_library(TSVConnFd SHARED TSVConnFd.cc)
+add_library(TSVConnFd MODULE TSVConnFd.cc)
 
 set_target_properties(TSVConnFd PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/.libs"
     PREFIX ""
 )
+target_link_libraries(TSVConnFd PRIVATE tscore traffic_server)
+set_target_properties( PROPERTIES PREFIX "")
+set_target_properties( PROPERTIES SUFFIX ".so")
```

Comment:
Does this work without the target name? Also, the `PREFIX` property is already set on line 22; the `SUFFIX` property should probably be set in the same `set_target_properties` command.